src/widgets/histogram/content-view.js

```diff
@@ -134,6 +138,7 @@ module.exports = cdb.core.View.extend({
 
     if (isDataEmpty) {
       this._addPlaceholder();
+      this._initTitleView();
```

Comment:
Why don't we add the title in the placeholder template? It could save us an unnecessary Backbone view.